src/main/java/core/basesyntax/SupplyOperation.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public class SupplyOperation implements OperationHandler {

```

Comment:
The class implementation shouldn't begin with an empty line according to the common mistakes checklist.